### DISCUSSION TITLE
Relate Apps to Workspaces

### DISCUSSION THREAD

**Discussion by mooster71** [migueesc123/PowerBIRESTAPI#172] [Q&A] [answered]
Really like this PBI API connector :)

While trying to put it to use I wondered how to relate Apps to Workspaces. I would like to know from which Workspace an App is published but the relation seems to be missing in the API exposed datasets?

Cheers,

Maarten

**Comment by klinejordan** [comment, marked as answer]
Technically Reports and Dashboards are related to Apps, not Workspaces. So that's why you'll see AppID as a foreign key in the Reports and Dashboards table but not in the Workspace table

**Reply by mooster71** [reply]
Thanks a lot! That makes perfect sense, should have thought of that myself...

Cheers!
